bin/plugin/commands/common.js

```diff
@@ -68,5 +82,26 @@ exports.getModuleData = async ( modulesDir ) => {
 
 			return moduleData;
 		} )
-		.filter( ( moduleData ) => moduleData.name && moduleData.description );
+		.filter( ( moduleData ) => moduleData.name && moduleData.description )
```

Comment:
See my comment above, it would be better to remove the above default and instead do the following extra validation here:
```suggestion
		.filter( ( moduleData ) => moduleData.name && moduleData.description && typeof moduleData.experimental !== undefined )
```

Reply:
Agree that this would work as well, however I see it like we have 2 options:

1. Initialize to a safe default in case the experimental value is missing in order to include a module.
2. Not include the file but create a warning that a file is missing a required property.

Removing a module by missing a header without an error message might introduce some silent errors if we are not being explicit either on the parameters or error message on the reason why a module is not included. 

Let me know what do you think about that :)

Reply:
@mitogh That's a good point. How about we expand the `filter` function to also trigger warnings? I think it's okay to not error, we can just ignore such modules technically, but at least there should be a corresponding console output with warnings.

In other words, I think we should go with my originally suggested idea in terms of conditions to check, but expand the function based on your idea that if any of the 3 conditions is not satisfied, there should be a warning triggered in the console. Probably most helpful would be to have a specific warning per which of the 3 checks went wrong.

Reply:
Great I think that's a good call, let me update this filter to include those warnings.